README.md

```diff
@@ -1,3 +1,48 @@
 [![Build Status](https://api.travis-ci.org/alphagov/paas-cf.svg)](https://travis-ci.org/alphagov/paas-cf)
 
 # paas-cf
+
+This repository contains [Concourse](http://concourse.ci/) pipelines and related [Terraform](https://terraform.io/) and [BOSH](https://bosh.io/) manifests that allow provisioning of [CloudFoundy](https://www.cloudfoundry.org/) on AWS. It consists of two chains:
+- A concourse pipeline to provision CloudFoundry on AWS using Concourse, Terraform and BOSH
+- A concourse pipeline to destroy previously provisioned environment.
+
+## Operation
```

Comment:
Maybe we can describe the pipelines in a standard way , with inputs, outputs and prerequisites, and  be more abstract (try to avoid implementation detauls)

Included pipelines
----------------------

 * `vpc-bootstrap`: Pipeline to bootstrap IaaS prerequisites like VPC and subnets using terraform.
    * Inputs: 
         * AWS credentials.
         * Deployment name.
    * Pre-requisites: 
        * Running concourse. A Vagrantfile to run concourse locally is provided.
    * Outputs: 
        * Terraform state.
        * IaaS setup

 * `concourse-deploy`: Deploys concourse on a given VPC
    * Inputs: 
         * AWS credentials passed as `fly` parameters
         * Deployment name
         * Terraform state from `bootstrap-vpc`
    * Pre-requests:
        * Running concourse. A Vagrantfile to run concourse locally is provided.
        * IaaS vpc and network pre-configured
    * Outputs:
        * Terraform state
        * Bosh init state
        * Running concourse server

 * `bosh-deploy`: Deploys a microbosh server using bosh-init 
    * Inputs: 
         * AWS credentials passed as `fly` parameters
         * Deployment name
         * Terraform state from `bootstrap-vpc`
    * Pre-requests:
        * Running concourse in the same target VPC.
        * IaaS vpc and network pre-configured
    * Outputs:
        * Terraform state
        * Bosh init state
        * Running bosh server

Etc...